connect/json/src/test/java/org/apache/kafka/connect/json/JsonConverterTest.java

```diff
@@ -848,7 +847,7 @@ public void noSchemaToJson() {
 
     @Test
     public void testCacheSchemaToJsonConversion() {
-        Cache<Schema, ObjectNode> cache = Whitebox.getInternalState(converter, "fromConnectSchemaCache");
+        Cache<Schema, ObjectNode> cache = converter.fromConnectSchemaCache;
         assertEquals(0, cache.size());
```

Comment:
Seems like we're just checking the size? Can we have a method for this only instead of exposing mutable fields?